flixpy/examples/revision_a_panel.py

```diff
@@ -0,0 +1,53 @@
+import asyncio
+
+import flix
+
+# Authentication data
+HOSTNAME = "localhost"
+PORT = 8080
+USERNAME = "admin"
+PASSWORD = "admin"
+
+# Flix IDs
+SHOW_ID = 1
+SEQUENCE_ID = 1
+SEQUENCE_REVISION = 1
+PANEL_ID = 1
+PANEL_REVISION = 1
+
+# New file to be used to create the new panel revision
+FILEPATH = "./new_file.psd"
+
+
+async def main() -> None:
+    async with flix.Client(HOSTNAME, PORT) as client:
+        # Log into the Flix server
+        await client.authenticate(USERNAME, PASSWORD)
+        # Get the show from the server
+        show = await client.get_show(SHOW_ID)
+        # Create an 'artwork' media object with the file and transcode it to create a thumbnail
+        with open(FILEPATH, 'rb') as f:
```

Comment:
```suggestion
        with open(FILEPATH, "rb") as f:
```

I'd suggest just running Black once

Reply:
black run after resolved comments, no further changes